app/src/main/java/com/amos/flyinn/settingsCheck/settingsCheck.java

```diff
@@ -0,0 +1,116 @@
+package com.amos.flyinn.settingsCheck;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.StrictMode;
+import android.provider.Settings;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.amos.flyinn.MainActivity;
+import com.amos.flyinn.summoner.ADBService;
+import com.tananaev.adblib.AdbBase64;
+import com.tananaev.adblib.AdbConnection;
+import com.tananaev.adblib.AdbCrypto;
+
+import java.io.Console;
+import java.io.IOException;
+import java.net.Socket;
+import java.util.HashSet;
+import java.util.List;
+
+import static android.support.v4.content.ContextCompat.startActivity;
+
+public class settingsCheck {
```

Comment:
Please add javadoc what this class is supposed to do. Also use CapitalCamelCase for class names.